src/main/java/TaskList.java

```diff
@@ -0,0 +1,119 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Manager to manage a list of tasks
+ */
+public class TaskList {
+    private final List<Task> ls = new ArrayList<>();
+    private final Storage storage;
+
+    TaskList(Storage storage){
+        this.storage = storage;
+    }
+
+    public void addTaskFromFile() throws IOException {
+        File file = this.storage.getFile();
```

Comment:
Maybe the conversion of File to Task can be done within the Storage class, so all TaskList has to deal with is Task